src/components/Blog/BlogArticlesBoard/BlogArticlesBoard.tsx

```diff
@@ -1,112 +1,223 @@
-import type { TabProps } from '@/components/Tabs';
-import type { BlogArticleData, TagAttributes } from '@/types/strapi';
-import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
-import type { Theme } from '@mui/material';
-import { Fade, Skeleton, useMediaQuery, useTheme } from '@mui/material';
+'use client';
+
+import { a11yProps, type TabProps } from '@/components/Tabs';
+import type { BlogArticleData, StrapiMetaPagination } from '@/types/strapi';
+import type { Breakpoint, Theme } from '@mui/material';
+import { Box, Skeleton, useMediaQuery, useTheme } from '@mui/material';
+import type { PropsWithChildren } from 'react';
 import { useCallback, useMemo, useState } from 'react';
 import { useTranslation } from 'react-i18next';
+import type { GetTagsResponse } from 'src/app/lib/getTags';
+import { TrackingCategory } from 'src/const/trackingKeys';
+import { BlogArticleCard } from '../BlogArticleCard';
+import { BlogArticleCardSkeleton } from '../BlogArticleCard/BlogArticleCardSkeleton';
 import {
+  ArticlesGrid,
   BlogArticlesBoardContainer,
   BlogArticlesBoardTitle,
 } from './BlogArticlesBoard.style';
 import { BlogArticlesBoardTabs } from './BlogArticlesBoardTabs';
+import { BlogArticlesBoardPagination as Pagination } from './Pagination';
 
 interface BlogArticlesBoardProps {
   data: BlogArticleData[];
-  tags: TagAttributes[];
+  tags: GetTagsResponse;
   url: string;
 }
 
+const ariaLabel = 'blog-articles-board-tabs';
```

Comment:
would be better to have something explicit like the title or what the action is doing